### DISCUSSION TITLE
Support for encoding in JPEG

### DISCUSSION THREAD

**Discussion by lemkoleg** [korlibs/korge#1316] [Q&A] [answered]
Hello. Is it planned to support image encoding in JPEG format. If yes, then when?

**Comment by soywiz** [comment, marked as answer]
It is possible already. Please, copy these files in your project: https://github.com/korlibs/korge/tree/8820c03f3b6c5d32410f6658d3ebea00a6c5493c/%40old/korim-jpeg/src/commonMain/kotlin/com/soywiz/korim/format/jpg

If using kproject, just include: https://github.com/korlibs/korge-image-formats/tree/main/korim-jpeg

Then:

```kotlin
RegisteredImageFormats.register(JPEG)
```
